src/components/TransactionsTable/TransactionsTable.tsx

```diff
@@ -106,11 +106,24 @@ const headCells = (
 
 const TransactionsTable: React.FC<TransactionsTableProps> = ({ data }) => {
   const [txFilter, setTxFilter] = useState('');
-  const tokenHeadCells = headCells(txFilter, setTxFilter);
+  const txHeadCells = headCells(txFilter, setTxFilter);
   const classes = useStyles();
   const { chainId } = useActiveWeb3React();
   const { palette, breakpoints } = useTheme();
   const isMobile = useMediaQuery(breakpoints.down('xs'));
+  const getTxString = (txn: any) => {
+    return `${txn.type} ${txn.pair.token0.symbol} ${
+      txn.type === 'Swap' ? 'for' : 'and'
+    } ${txn.pair.token1.symbol}`;
+  };
+  const getTxAmount0String = (txn: any) => {
+    const token = txn.type === 'Swap' ? txn.pair.token1 : txn.pair.token0;
+    return `${formatNumber(txn.amount0)} ${token.symbol}`;
+  };
+  const getTxAmount1String = (txn: any) => {
```

Comment:
There's no need for 2 methods for this just pass the amount in.

Reply:
We are showing different token for 2 amount columns in transactions table on analytics pair details. That's why I created 2 functions for each amount column

Reply:
You don't gain much by calling the method 0 or 1 just pass the amount in instead.

const getTxAmountString = (txn: any,  amount: Number) => {
    const token = txn.type === 'Swap' ? txn.pair.token1 : txn.pair.token0;
    return `${formatNumber(amount)} ${token.symbol}`;
  };

Reply:
Moved transaction messages to `en.json` for localization and pushed updates